src/Castle.Core/DynamicProxy/Generators/Emitters/SimpleAST/Reference.cs

```diff
@@ -47,12 +47,7 @@ public virtual void Generate(ILGenerator gen)
 
 		public void Emit(IMemberEmitter member, ILGenerator gen)
 		{
-			this.EmitLoadOwnerAndReference(gen);
-		}
-
-		public void EmitLoadOwnerAndReference(ILGenerator gen)
-		{
-			OwnerReference?.EmitLoadOwnerAndReference(gen);
+			OwnerReference?.Emit(member, gen);
 			LoadReference(gen);
 		}
```

Comment:
I'm not entirely sure that nothing important is lost (in terms of semantics) when the (outward) distinction between `Emit` and `EmitLoadOwnerAndReference` is erased.

(It's true that `reference.Emit` always did the same as `EmitLoadOwnerAndReference`, but because it appears to never have been called directly, call sites read differently up until now.)